src/dataflow/operators/to_stream.rs

```diff
@@ -4,7 +4,7 @@ use progress::Timestamp;
 
 use Data;
 use dataflow::channels::Content;
-use dataflow::operators::operator::source;
+use dataflow::operators::generic::operator::source;
```

Comment:
It could be; I don't think it is re-exported in `generic` yet, but we could do that (to hide where it lives, perhaps)?